### PR TITLE
feat(terser): emit source map

### PR DESCRIPTION
**Pull request by tada5hi**
## Rollup Plugin Name: `terser`

This PR contains:

- [ ] bugfix
- [x] feature
- [ ] refactor
- [ ] documentation
- [ ] other

Are tests included?

- [x] yes (_bugfixes and features will not be merged without tests_)
- [ ] no

Breaking Changes?

- [ ] yes (_breaking changes will not be merged unless absolutely necessary_)
- [x] no

If yes, then include "BREAKING CHANGES:" in the first commit message body, followed by a description of what is breaking. 

List any relevant issue numbers:

### Description

feat #1371

### CODE REVIEW

**Comment by tada5hi** [comment]
@shellscape this is a bit non related here.
But can you take a look at the `codeowner` file in the root directory.
It still complains about missing permissions.

**Comment by shellscape** [comment]
what complains about what?

**Comment by tada5hi** [comment]
I'm not quite sure. If you open the file in GitHub, it displays: "This CODEOWNERS file contains errors"
When you hover over the marked line it shows the following dialogue:
`Unknown owner: make sure @tada5hi exists and has write access to the repository`

**Comment by shellscape** [comment]
wouldn't worry about it right now. you should get notified of PRs against the plugin.

**Comment by tada5hi** [comment]
@shellscape would be great if you could have a look. I think the pr can be merged :v:

**Comment by tada5hi** [comment]
@shellscape let me know if something is missing ✌️.

**Review by shellscape — APPROVED** [review]

File: packages/terser/test/test.js
Comment (shellscape) [inline packages/terser/test/test.js:30]: just caught this after clicking approve: an empty array will always be truthy. you want to check the length. a snapshot might be a good idea here.

**Review by shellscape — CHANGES_REQUESTED** [review]

File: packages/terser/test/test.js
Comment (tada5hi) [reply packages/terser/test/test.js]: yep, you are completly right. While writing the test, the `deepEqual` operator did not occur to me, because `is` can not be used for array comparision.
I fixed the line.

**Comment by tada5hi** [comment]
@shellscape can you have a look again ?

**Review by shellscape — APPROVED** [review]

**Pull request merged**
